flexget/utils/titles/series.py

```diff
@@ -245,98 +244,87 @@ def parse(self, data=None, field=None, quality=None):
         data_stripped = data_stripped.lower()
         log.debug('data stripped: %s', data_stripped)
 
-        # allow group(s)
-        if self.allow_groups:
-            for group in self.allow_groups:
-                group = group.lower()
-                for fmt in ['[%s]', '-%s']:
-                    if fmt % group in data_stripped:
-                        log.debug('%s is from group %s', self.data, group)
-                        self.group = group
-                        data_stripped = data_stripped.replace(fmt % group, '')
-                        break
-                if self.group:
-                    break
-            else:
-                log.debug('%s is not from groups %s', self.data, self.allow_groups)
-                return  # leave invalid
-
-        # Find quality and clean from data
-        log.debug('parsing quality ->')
-        quality = qualities.Quality(data_stripped)
-        if quality:
-            # Remove quality string from data
-            log.debug('quality detected, using remaining data `%s`', quality.clean_text)
-            data_stripped = quality.clean_text
-        # Don't override passed in quality
-        if not self.quality:
-            self.quality = quality
-
         # Remove unwanted words from data for ep / id parsing
         data_stripped = self.remove_words(data_stripped, self.remove, not_in_word=True)
 
+        groups_data_stripped = data_stripped
+
         data_parts = re.split('[\W_]+', data_stripped)
 
         for part in data_parts[:]:
             if part in self.propers:
                 self.proper_count += 1
                 data_parts.remove(part)
+                groups_data_stripped = groups_data_stripped.replace(part, '')
             elif part == 'fastsub':
                 # Subtract 5 to leave room for fastsub propers before the normal release
                 self.proper_count -= 5
                 data_parts.remove(part)
+                groups_data_stripped = groups_data_stripped.replace(part, '')
             elif part in self.specials:
                 self.special = True
                 data_parts.remove(part)
+                groups_data_stripped = groups_data_stripped.replace(part, '')
 
         data_stripped = ' '.join(data_parts).strip()
```

Comment:
What is the difference between data_stripped and groups_data_stripped at this point?